content/blog/2019/02/2019-02-08-let-s-celebrate-java-11-support.adoc

```diff
@@ -77,7 +77,7 @@ From there, we will port the support to the link:https://github.com/jenkins-x[Je
 We also have to work on the link:projects/evergreen/[Evergreen] project support for Java 11.
 
 So, even if this is a big deal to us, this is not the end of the story.
-It is more the first step which you, users, developers, members of this community, can really benefit from.
+It a major step that will benefit users, developers, and members of the Jenkins community.
```

Comment:
Sorry, I missed a word in the earlier suggestion.

> It a major step that will benefit users

should be

> It is a major step that will benefit users

Reply:
So, even though this is a big deal for us, and a major improvement that benefits users, developers and members of the Jenkins community...it's not the end of the story.

Reply:
```suggestion
It is a major step that will benefit users, developers, and members of the Jenkins community.
```